ext/bcmath/libbcmath/src/doaddsub.c

```diff
@@ -166,7 +163,56 @@ bc_num _bc_do_sub(bc_num n1, bc_num n2, size_t scale_min)
 	}
 
 	/* Now do the equal length scale and integer parts. */
-	for (count = 0; count < min_len + min_scale; count++) {
+	size_t sub_count = 0;
```

Comment:
Analyzing the measurement results, it seems that sharing the count here and resetting it once is more expensive than using a separate variable.

Reply:
That seems odd, did you check via hyperfine? Perhaps that's just coincidental noise, hyperfine can filter that out.

Reply:
I think I measured it using hyperfine, but I'll separate the commits and measure them again individually.

Reply:
Not much changed. In fact, it was a little late. Therefore, I will not make this change.